src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -19,23 +19,65 @@ import '../TextBlock/TextBlock';
 import '../TextRun/TextRun';
 
 /**
- * AdaptiveCard component
+ * AdaptiveCardInternal component
  *
  * @param {object} props  React properties
  * @param {object} props.data  Active Cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
  * @returns {object} JSX of the component
  */
-export default function AdaptiveCard({data}) {
-  const [cssClasses] = webexComponentClasses('adaptive-card', 'wxc-ac-container--has-padding wxc-ac-container--vertical');
+function AdaptiveCardInternal({data, className, style}) {
+  const [cssClasses] = webexComponentClasses('adaptive-card', className);
```

Comment:
can you please modify `webexComponentClasses` to accept an array of classes as the second param?

```js
const [cssClasses] = webexComponentClasses('adaptive-card', [className, 'wxc-ac-container--has-padding']);

Reply:
https://www.npmjs.com/package/classnames#user-content-usage

Because `classNames()` can take a variable number of parameters, I would suggest this implementation:

```js
  const userClassNames = Array.isArray(userClassName) ? userClassName : [userClassName];

  const cssClasses = classNames(
    `${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}`,
    ...userClassNames,
    {
      ...Object.fromEntries(
        Object.entries(otherClasses || {})
          .map(([key, val]) => [`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}--${key}`, val]),
      ),
    }
  );